catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/solr/impl/CachingFederationStrategyTest.java

```diff
@@ -85,39 +90,35 @@
 
   private ExecutorService cacheExecutor, queryExecutor;
 
-  private Query mockQuery;
+  @Mock private Query mockQuery;
 
   private CachingFederationStrategy strategy;
 
-  private PreFederatedQueryPlugin preQueryPlugin;
+  private CachingFederationStrategy federateStrategy;
 
-  private MetacardImpl mockMetacard;
-
-  private Result mockResult;
+  @Mock private PreFederatedQueryPlugin preQueryPlugin;
 
-  private SourceResponse mockResponse;
+  private MetacardImpl mockMetacard;
```

Comment:
✏️  I wouldn't put `mock` in front of the name of this object because it's technically a real impl.